### DISCUSSION TITLE
REQUEST - ADD SHIPPING CARRIER SUPPORT - OrangeDS

### DISCUSSION THREAD

**Discussion by kiernangs** [etsy/open-api#1162] [Ideas]
I am requesting that etsy add tracking support for carrier OrangeDS

Tracking page:
https://tracking.orangeds.com/

Tracking number examples with aftership
https://www.aftership.com/track/orangedsinc/ODSAGA00000034251581
https://www.aftership.com/track/orangedsinc/ODSAGA00000034251589
https://www.aftership.com/track/orangedsinc/420380189261290172381001473448

----
They are already supported by aftership which etsy says makes carrier additions easier

**Comment by Gareth064** [comment]
Hey @kiernangs - Etsy developers don't really monitor this.

I'd email them developer@etsy.com and point them to this request here and ask them if they can do it.

**Comment by kiernangs** [comment]
Anyone know of any actual method remaining to ever get etsy to care and add carrier?

Developer email  gets canned replies.
Support chat barely knows how to answer basic questions and cant comprehend the idea of other carriers "We suggest using a supported carrier" (thanks...)

And etsy devs supposedly no longer look at this forum?

**Comment by kiernangs** [comment]
@nateswart  Are you still able to add carriers that are directly supported by aftership? Because OrangeDS is supported by aftership as shown above.

The etsy fulfilment api still links here for sending feedback regarding new carriers - if there is another method please let me know (https://developer.etsy.com/documentation/tutorials/fulfillment/#tracking-updates-for-shipping-carriers)

**Reply by Nikolasgrizli** [reply]
I would like to join your request. I'm trying to find a way to add our service to the list of carriers, but in response to my numerous requests, i only get a response like ‘we do not have a team to add new carriers’
